_data/cryptocurrencies.yml

```
@@ -211,6 +211,7 @@ websites:
       tfa: Yes
       sms: Yes
       software: Yes
+      hardware: Yes
       phone: Yes
       doc: https://support.coinbase.com/customer/en/portal/articles/1658338-how-do-i-set-up-2-factor-authentication-
```

Let's compress that documentation URL :wink:
```suggestion
      doc: https://support.coinbase.com/customer/en/portal/articles/1658338
```